packages/fiber/tests/setupTests.ts

```diff
@@ -1,5 +1,55 @@
-import { extend } from '../src'
+import { createWebGLContext } from '@react-three/test-renderer/src/createWebGLContext'
 import * as THREE from 'three'
+import { extend } from '../src'
+import * as React from 'react'
+import { ViewProps, LayoutChangeEvent } from 'react-native'
+
+// Mock scheduler to test React features
+jest.mock('scheduler', () => require('scheduler/unstable_mock'))
+
+// Polyfills WebGL canvas
+// @ts-expect-error
+HTMLCanvasElement.prototype.getContext = function () {
+  return createWebGLContext(this)
+}
 
 // Extend catalogue for render API in tests
 extend(THREE)
+
+// Mock native dependencies for native
+jest.mock('react-native', () => ({
+  StyleSheet: {},
+  View: React.memo(
+    React.forwardRef(({ onLayout, ...props }: ViewProps, ref) => {
+      React.useLayoutEffect(() => {
+        onLayout?.({
+          nativeEvent: {
+            layout: {
+              x: 0,
+              y: 0,
+              width: 1280,
+              height: 800,
+            },
+          },
+        } as LayoutChangeEvent)
+
+        ref = { current: { props } }
+      }, [])
+
+      return null
+    }),
+  ),
+}))
```

Comment:
Will need to configure jest to transform RN/flow so we can drop these mocks and assert against the native react tree.

Reply:
Dropped the Pressability lib for native's Pressable so we can better test events/props since they're not converted into low-level handlers. Keeping the Canvas typed as a View since we'll overwrite Pressable props ourselves.

When RTL updates native to react 18, we can look at the react tree and know that the canvas is mounted and all of the events are there.